src/context/client.tsx

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
```

Comment:
no biggie but just to check, could refactoring to use the axios client have happened independently of the "login ui" or are they intertwined?

Reply:
Refactoring to use axios can happen independently. It's more of a dependency for the UI work.